src/convenience/frameworks.ts

```diff
@@ -239,7 +243,7 @@ export type WorktopAdapter = (req: {
 
 /** AWS lambda serverless functions */
 const awsLambda: LambdaAdapter = (event, _context, callback) => ({
-    update: JSON.parse(event.body ?? "{}"),
+    update: Promise.resolve(JSON.parse(event.body ?? "{}")).catch(empty),
```

Comment:
What is your intent here? This looks a bit like you're trying to catch JSON parse errors, but this is not what is happening here

Reply:
i think you are right, my bad